framework/scripts/restart_drill_script

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+source ./conf/drillTestConfig.properties
+set -x
+clush -a ${DRILL_HOME}/bin/drillbit.sh restart
```

Comment:
The framework change is meant to provide a hook for calling custom script which will do the real job based on custom environment. This is just a sample implementation. I or someone can work on a more complete implementation in another PR. Maybe a different version for Docker setup too.